### PR TITLE
Add epoch support for time helper function

### LINKED ISSUE #87: Support for epochs
There are some APIs that return different timestamps using epoch (ZMON's is one of these, to give an example).  It would be nice to have the time() helper function handling these.  One further improvement could possibly be adding the datetime.strptime() functionality to make life easier.

### PR DESCRIPTION
**Pull request by mohabusama**
Fixes #87

### CODE REVIEW

File: zmon_worker_monitor/builtins/plugins/time_.py [hunk 1]
```diff
@@ -29,15 +29,19 @@ def create(self, factory_ctx):
 
 
 class TimeWrapper(object):
-    def __init__(self, spec='now', utc=False):
+    def __init__(self, spec='now', epoch=None, utc=False):
```
Comment (hjacobs) [inline zmon_worker_monitor/builtins/plugins/time_.py]: why not make it a single argument? using time(123123) looks much simpler than a keyword arg

We could simply assume every int/long is epoch :smile:

Comment (mohabusama) [inline zmon_worker_monitor/builtins/plugins/time_.py]: Done.

**Comment by hjacobs** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

**Pull request merged**